src/coreComponents/mesh/generators/VTKMeshGenerator.cpp

```diff
@@ -150,56 +176,100 @@ loadMesh( Path const & filePath )
   return loadedMesh;
 }
 
+vtkNew< vtkCellArray > GetCellArray( vtkDataSet & mesh ) // replaces GetCells() that exist only in vtkUnstructuredGrid
+{
+  vtkNew< vtkCellArray > cells;
+  if( mesh.IsA( "vtkStructuredGrid" ) || mesh.IsA( "vtkUnstructuredGrid" ))
+  {
+    int numCell = mesh.GetNumberOfCells();
+    for( int c = 0; c < numCell; c++ )
+    {
+      cells->InsertNextCell( mesh.GetCell( c ));
+    }
+  }
+  else
+  {
+    double bounds[6];
+    mesh.GetBounds( bounds );
+    double xmin=bounds[0], xmax=bounds[1];
```

Comment:
`zmin` and `zmax` are `double`s or are they always integers? If they are always integers, please declare them as integers and round and cast the values of `bounds` (see comment below). This would make the loops at line 201 more readable.